Tests/SwiftLintFrameworkTests/RulesTests.swift

```diff
@@ -96,11 +96,6 @@ class RulesTests: XCTestCase {
         verifyRule(FatalErrorMessageRule.description)
     }
 
-    func testFileLength() {
```

Comment:
This was added to `FileLenghtRuleTests`: https://github.com/realm/SwiftLint/pull/1702/files#diff-d2c68a7ad0b1aca075f32e032d5ea03aR14